examples/evm/its-lock-unlock-fee/FeeToken.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { InterchainTokenStandard } from '@axelar-network/interchain-token-service/contracts/interchain-token/InterchainTokenStandard.sol';
+import { ERC20 } from '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+
+import { Minter } from '@axelar-network/interchain-token-service/contracts/utils/Minter.sol';
+
+/**
+ * @title FeeToken
+ * @notice This contract implements an interchain token which extends InterchainToken functionality and extracts a fee for each transfer.
+ * @dev This contract also inherits Minter and Implementation logic.
+ */
+contract FeeToken is InterchainTokenStandard, ERC20, Minter {
+    uint8 internal immutable decimals_;
```

Comment:
Is this postfix/prefix notation consistent across our docs? I thought it was the other way around sometimes.

Reply:
Will double check this^

Reply:
Sorry, just to clarify, do you mean for the decimals_ variable here if `uint8 internal immutable decimals_` should be prefix? A bit unclear on this. Thanks!

Reply:
I think there should be no postfixes, only prefixes. This will keep us consistent with all of our other code samples.

Example: https://github.com/axelarnetwork/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol#L26

Reply:
Oh! I see what you mean now, ya i'll change this good catch